src/main/java/org/synyx/urlaubsverwaltung/config/RestApiSecurityConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.synyx.urlaubsverwaltung.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+
+@Configuration
+@Order(1)
```

Comment:
Hier lieber `SecurityProperties.BASIC_AUTH_ORDER-1` nehmen? Dann sind wir relative.

Reply:
/**
	 * Order applied to the WebSecurityConfigurerAdapter that is used to configure basic
	 * authentication for application endpoints. If you want to add your own
	 * authentication for all or some of those endpoints the best thing to do is to add
	 * your own WebSecurityConfigurerAdapter with lower order.
	 */

Reply:
die `SecurityProperties.BASIC_AUTH_ORDER-1` ist leider viel zu weit hinten. Unsere normale Config hat ein Value von 100 und die RestApiSecurityConfig.java muss vorher greifen, sonst werden wir auf mit einem Http Status 302 auf /login redirected